gradle/javadoc.gradle

```diff
@@ -17,11 +17,11 @@ import static org.gradle.util.CollectionUtils.single
  */
 
 
-def javaMainProjects = subprojects.findAll { !['util'].contains(it.name) }
+def nonDocsProjects = project(":util").allprojects + project(":driver-benchmarks") + project("driver-workload-executor")
```

Comment:
This variable name confused me.  I was thinking it would include driver-core, bson, etc, based on the name.  It's more like nonDocsOrMainProjects?    Maybe we should just inline it...

Reply:
These are all the projects for which we don't publish javaDoc or ScalaDocs for.

Changed to: `projectsThatDoNotPublishJavaDocs`